lib/tasks/scheduler.rake

```diff
@@ -3,10 +3,10 @@
 # License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
 # Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
 
-desc "For use with Heroku's Scheduler add-on"
+desc "Task used to run any ScheduledJob"
 
 # We use a single rake call so we can catch and send any errors that happen in the job
-task :heroku_scheduled_job, [:name] => :environment do |_t, args|
+task :cron_job_runner, [:name] => :environment do |_t, args|
```

Comment:
So I just realized that we should still have an heroku_scheduled_job task. Some people probably depend on the original being there.

You should readd a heroku_scheduled_job task, have it only run the cron_job_runner and in it's description mention that it's deprecated and is an alias for cron_job_runner.

Reply:
Done!